internal/cmd/run.go

```diff
@@ -87,3 +105,26 @@ func initializeRunOpts(o *globals.GlobalOpts, platform, version string) error {
 	}
 	return nil
 }
+
+// setHomeEnvoyVersion makes sure the $GETENVOY_HOME/version exists and globals.GlobalOpts HomeEnvoyVersion contains it.
+func setHomeEnvoyVersion(o *globals.GlobalOpts) error {
```

Comment:
Prefer have this function as a method of `globals.GlobalOpts`.. WDYT?

Reply:
so far I've not had any cmd arg parsing in the globals file.. that's why this is here..

Reply:
OK, got it